docs/adguard-for-android/solving-problems/samsungpay-with-adguard-in-south-korea.md

```diff
@@ -3,7 +3,7 @@ title: How to use Samsung Pay with AdGuard in South Korea
 sidebar_position: 14
 ---
 
-South Korean users often face the issue that Samsung Pay doesn't work on devices with running VPN services. Luckily, there is a way to solve this problem.
+This problem occurs almost exclusively on devices registered in South Korea. For such devices, Samsung Pay sometimes doesn't work on devices with running VPN services. However, there is a way to solve this problem.
```

Comment:
This problem occurs almost exclusively on devices registered in South Korea. Samsung Pay occasionally doesn't work on such devices with running VPN services. However, there is a way to solve this problem.